sdk/deepbook/src/client.ts

```diff
@@ -329,12 +377,12 @@ export class DeepBookClient {
 	 */
 	async swapExactQuoteForBase(
 		poolId: string,
-		tokenObjectIn: string,
-		amountIn: bigint,
+		tokenObjectIn: TransactionResult | TransactionObjectArgument | string,
+		amountIn: bigint, // quantity of USDC
```

Comment:
this method only works for USDC?

Reply:
Yea, it is to swap quote (USDC) with base (SUI). The pool is fixed on base/quote when initiating the first time